src/main/java/com/exasol/containers/ExasolContainer.java

```diff
@@ -596,6 +604,28 @@ private void clusterConfigurationIsAvailable() {
         }
     }
 
+    private static final int MAJOR_DEPRECATED_VERSION = 6;
+    private static final int MINOR_DEPRECATED_VERSION = 2;
+
+    private void ThrowDBVersionNotSupportedException() {
+        throw new ContainerLaunchException(
+                ExaError.messageBuilder("E-ETC-13")
+                        .message("Exasol Database version " + MAJOR_DEPRECATED_VERSION + "." + MINOR_DEPRECATED_VERSION
+                                + " and lower are no longer supported in this version of Exasol Testcontainers.")
+                        .toString());
+    }
+
+    private void CheckClusterConfigurationForMinimumSupportedDBVersion() {
+        // do the check
+        String dbVersion = clusterConfiguration.getDBVersion();
+        String[] dbVersionSplit = dbVersion.split(Pattern.quote("."));
```

Comment:
validate that version wath split in >= 2 parts. Otherwise NullPointer exception could happen when accessing

Reply:
version should always be there, so ...

Reply:
added a check

Reply:
and a mitigation